public/app/features/alerting/unified/home/GettingStarted.tsx

```diff
@@ -38,107 +37,72 @@ export default function GettingStarted({ showWelcomeHeader }: { showWelcomeHeade
             <li>Firing instances are routed to notification policies based on matching labels</li>
             <li>Notifications are sent out to the contact points specified in the notification policy</li>
           </ul>
-        </div>
-        <SVG
-          src={`public/img/alerting/at_a_glance_${theme.name.toLowerCase()}.svg`}
-          width={undefined}
-          height={undefined}
-        />
+          <div className={styles.svgContainer}>
+            <Stack justifyContent={'center'}>
+              <SVG
+                src={`public/img/alerting/at_a_glance_${theme.name.toLowerCase()}.svg`}
+                width={undefined}
+                height={undefined}
+              />
+            </Stack>
+          </div>
+        </Stack>
       </ContentBox>
-      <ContentBox className={styles.gettingStartedBlock}>
-        <h3>Get started</h3>
-        <Stack direction="column">
+      <ContentBox>
+        <Stack direction="column" gap={1}>
+          <Text element="h3">Get started</Text>
           <ul className={styles.list}>
             <li>
-              <strong>Create an alert rule</strong> by adding queries and expressions from multiple data sources.
+              <Text weight="bold">Create an alert rule</Text> by adding queries and expressions from multiple data
+              sources.
             </li>
             <li>
-              <strong>Add labels</strong> to your alert rules <strong>to connect them to notification policies</strong>
+              <Text weight="bold">Add labels</Text> to your alert rules{' '}
+              <Text weight="bold">to connect them to notification policies</Text>
             </li>
             <li>
-              <strong>Configure contact points</strong> to define where to send your notifications to.
+              <Text weight="bold">Configure contact points</Text> to define where to send your notifications to.
             </li>
             <li>
-              <strong>Configure notification policies</strong> to route your alert instances to contact points.
+              <Text weight="bold">Configure notification policies</Text> to route your alert instances to contact
+              points.
             </li>
           </ul>
-          <div>
-            <ArrowLink href="https://grafana.com/docs/grafana/latest/alerting/" title="Read more in the Docs" />
-          </div>
+          <TextLink href="https://grafana.com/docs/grafana/latest/alerting/" icon="angle-right" inline={false}>
```

Comment:
Sadly we can't use `TextLink` for external refs unless `external={true}`, that will render a proper `<a>` tag, otherwise the URL is passed in to the regular react-router-dom `<Link>` which doesn't support arbitrary http(s) hrefs :(

Reply:
Oops - thanks! Fixed this. 

I was going to add a test for this (to make sure that `TextLink` with an external link but no `external` prop set would throw an error), but[ there's some existing logic](https://github.com/grafana/grafana/pull/81799) to handle external URLs and stripping base URLs etc. and I thought it best not to complicate the component any further?